.github/workflows/validate.yml

```diff
@@ -0,0 +1,31 @@
+name: ValidateTrialJsonSchema
+on:
+  push:
+    branches: [ feature/json-validation ]
+  pull_request:
+    branches: [ feature/json-validation ]
+
+
+jobs:
+  Validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Check changed files
+        id: calc_files_diff
+        run: |
+          git fetch origin main:localmain
+
+          export FILES=$(git diff --name-only localmain | grep .json | grep -v "trial-json-schema")
```

Comment:
This is the line in which the grep -v indicates to ignore the schema file

Reply:
like

Reply:
Added in your workflow

Reply:
here too, a comment to explain.